presto-docs/src/main/sphinx/security/server.rst

```diff
@@ -151,6 +156,16 @@ See :doc:`/develop/system-access-control` for details.
 
 .. _coordinator-troubleshooting:
 
+User Mapping
+------------
+
+After authenticating with Kerberos, the Presto server receives the user's principal which is typically similar to
+an email address.  For example, when ``alice`` logs in in Presto might receive ``alice@example.com``.  By default,
+Presto will use the full Kerberos principal name, but this can be mapped to a shorter name using an user-extraction
+pattern.  For simple extraction rules the  ``http-server.authentication.krb5.user-extraction.pattern`` configuration
+property can be set to Java regular expression, and Presto will use the value of the first matcher group.  If the
+regular expression do does not match, the authentication is denied.  For more complex user-extraction rules see .
```

Comment:
Extra word "do"

Reply:
This last sentence seems wrong (mentions "extraction") and doesn't link anywhere